.circleci/config.yml

```diff
@@ -28,20 +28,6 @@ aliases:
         mv /tmp/docker/* /usr/bin
 
 commands:
-  emit_deployment_event:
```

Comment:
Cool with me but wanted to make sure this was intentional?

Reply:
Yes, this deployment event does not exist in this repo